src/components/families/family-sidebar/family-sidebar-enrolment-card/FamilySidebarEnrolmentCard.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { useState } from "react";
+
+import { Box, Divider, Typography } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import {
+  KeyboardArrowDownRounded,
+  KeyboardArrowUpRounded,
+} from "@material-ui/icons";
+
+import StatusChip from "components/common/status-chip/StatusChip";
+import EnrolmentStatus from "constants/EnrolmentStatus";
+import { Student } from "types";
+
+const useStyles = makeStyles((theme) => ({
+  card: {
+    backgroundColor: theme.palette.backgroundSecondary.default,
+  },
+  content: {
+    "& .MuiTypography-body2": {
+      color: theme.palette.text.secondary,
+      fontSize: 12,
+      fontWeight: 500,
+    },
+  },
+  session: {
+    "& .MuiTypography-body2": {
+      fontWeight: 700,
+    },
+  },
+  days: {
+    "& .MuiTypography-body2": {
+      fontWeight: 700,
+      color: "#0052CC", // not sure if this colour is used anywhere else in the app, should add to theme if yes (taken from figma)
+    },
+  },
+  divider: {
+    padding: "0.5px",
+    margin: theme.spacing(1),
+    background: theme.palette.text.secondary,
+  },
+}));
+
+type Props = {
+  session: string;
+  days: string;
+  status: EnrolmentStatus;
+  studentIDs: number[];
+  students: Student[];
+};
+
+const FamilySidebarEnrolmentCard = ({
+  session,
+  days,
+  status,
+  studentIDs,
+  students,
+}: Props) => {
+  const [showStudents, setShowStudents] = useState(false);
+  const classes = useStyles();
+
+  return (
+    <Box
+      onClick={() => setShowStudents(!showStudents)}
```

Comment:
instead of using an onClick here, it might be more more intuitive to users if there's a button to expand! there's a nice example on the mui card component [here](https://material-ui.com/components/cards/#complex-interaction), which also uses the Collapse component for smooth open/close animations!